### PR TITLE
Interpolation docs

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

File: docs/interpolation.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.. _interpolation:
+
+Interpolation of IRFs
+=====================
+
+This module contains functions to interpolate from a set of IRFs for different
+condiditions to a new IRF.
```
Comment (HealthyPear) [inline docs/interpolation.rst:7]: small typo

File: docs/interpolation.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+This can e.\,g. be used to interpolate IRFs for zenith angles of 20° and 40°
+to 30°.
```
Comment (jsitarek) [inline docs/interpolation.rst:10]: Hi @maxnoe 

I think it should me mentioned in the documentation that the interpolation is done over two dimensions (e.g. zenith and azimuth).

And sorry for forgetting the old compare_irf_cuts function - thx for fixing it here.

Comment (maxnoe) [reply docs/interpolation.rst:10]: It *could* be done using two dimensions, however, the functions allow arbitrary number of dimensions, right?

File: docs/interpolation.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.. _interpolation:
+
+Interpolation of IRFs
+=====================
+
+This module contains functions to interpolate from a set of IRFs for different
+condiditions to a new IRF.
+
+This can e.\,g. be used to interpolate IRFs for zenith angles of 20° and 40°
```
Comment (HealthyPear) [inline docs/interpolation.rst:9]: why the comma?

**Review by HealthyPear — CHANGES_REQUESTED** [review]

File: docs/interpolation.rst (hunk 3 above)
Comment (maxnoe) [reply docs/interpolation.rst:9]: Sorry, LaTeX muscle memory. This inserts a small non-breaking space in LaTeX, which is the proper way to typeset abbreviations.

File: docs/interpolation.rst (hunk 2 above)
Comment (HealthyPear) [reply docs/interpolation.rst:10]: is "IRFsaa" a special thing I don't know or a typo?

Comment (maxnoe) [reply docs/interpolation.rst:10]: typo :see_no_evil:

Comment (jsitarek) [reply docs/interpolation.rst:10]: Hi @maxnoe 
bulk of the functions is done in a way that it would allow an arbitrary number of dimensions, but the actual function used for the interpolation: griddata I think works only in 2D. Since this is the most typical use case, I think this is not a problem, but we can also consider substituting the griddata function with some other interpolation engine for higher number of dimensions.

File: pyirf/spectral.py [hunk 4]
```diff
@@ -242,6 +245,8 @@ class TableInterpolationSpectrum:
     By default, flux is interpolated linearly in log-log space.
     '''
     def __init__(self, energy, flux, log_energy=True, log_flux=True, reference_energy=1 * u.TeV):
+        '''Create a new TableInterpolationSpectrum spectrum'''
+        super().__init__(normalization=normalization, index=index, e_ref=e_ref)
```
Comment (jsitarek) [inline pyirf/spectral.py:249]: this does not seem to be related to the PR, and also the commit says only "Add docstring for __init__ methods ", while this last line does something different, was it intended ?

Comment (maxnoe) [reply pyirf/spectral.py:249]: Good catch, I unintentionally copied two lines.

File: docs/interpolation.rst (hunk 2 above)
Comment (maxnoe) [reply docs/interpolation.rst:10]: https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.griddata.html

No, griddata does arbitrary number of dimensions

**Comment by maxnoe** [comment]
Mmh. I cannot reproduce the three warnings locally

**Comment by maxnoe** [comment]
Ha, now it works.

**Review by HealthyPear — APPROVED** [review]

**Review by HealthyPear — APPROVED** [review]
